### PR TITLE
Updated notes

### PR DESCRIPTION
**Pull request by weitzhandler**
See #2853.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@weitzhandler : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

File: docs/ide/class-designer/how-to-add-class-diagrams-to-projects.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ In **Solution Explorer** or Class View, right-click the project and choose **Vie
 
 An auto-populated class diagram is created.
 
+## Notes
+The class designer is not available in .NET Core projects.
```
Comment (gewarren) [inline docs/ide/class-designer/how-to-add-class-diagrams-to-projects.md]: For consistency, please format it as follows (and remove line 69 `## Notes`).

```suggestion
> [!NOTE]
> Class Designer is not available in .NET Core projects.
```

**Comment by PRMerger17** [comment]
@weitzhandler : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@gewarren - let us know if this is ready by adding `#sign-off` . Thanks!

File: docs/ide/class-designer/how-to-add-class-diagrams-to-projects.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ In **Solution Explorer** or Class View, right-click the project and choose **Vie
 
 An auto-populated class diagram is created.
 
+## Notes
```
Comment (gewarren) [inline docs/ide/class-designer/how-to-add-class-diagrams-to-projects.md]:
```suggestion
```

**Comment by PRMerger10** [comment]
@weitzhandler : Thanks for your contribution! The author, @gewarren, has been notified to review your proposed change.

**Pull request merged**